service/src/test/resources/ehr/mapper/encounter/example-encounter-resource-15.json

```diff
@@ -0,0 +1,79 @@
+{
+  "resourceType":"Encounter",
+  "id":"F550CC56-EF65-4934-A7B1-3DC2E02243C3",
+  "meta":{
+    "profile":[
+      "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-Encounter-1"
+    ]
+  },
+  "identifier":[
+    {
+      "system":"https://EMISWeb/A82038",
+      "value":"F550CC56-EF65-4934-A7B1-3DC2E02243C3"
+    }
+  ],
+  "status":"finished",
+  "type":[
+    {
+      "text":"GP Surgery",
+      "coding": [
+        {
+          "system":"http://snomed.info/sct",
+          "code":"24561000000109",
+          "display":"A+E report"
+        }
+      ]
+    }
+  ],
+  "subject":{
+    "reference":"Patient/88F14BF6-CADE-47D6-90E2-B10519BF956F"
+  },
+  "participant":[
+    {
+      "type":[
+        {
+          "coding":[
+            {
+              "system":"http://hl7.org/fhir/v3/ParticipationType",
+              "code":"PPRF",
+              "display":"primary performer"
```

Comment:
I may have misunderstood the test but
Believe this file is used as INPUT_JSON_WITHOUT_PERFORMER_PARTICIPANT in EncounterMapperTest
But this resource has participant of type performer?

Reply:
Well, that was incredibly dumb of me. Tests passed because we're mocking everything to have the same ID. Now fixed